# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/ds/StaticAtoms.py
# Commit: f50ff29a1a64
# Full Hash: f50ff29a1a649e3aad723b592bdd2c3c325b3d99
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 1: Screen Wake Lock boilerplate. r=dom-core,smaug,emilio,edgar
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189508
# ==============================================================================

diff -r 00afe2e46b1b -r f50ff29a1a64 xpcom/ds/StaticAtoms.py
--- a/xpcom/ds/StaticAtoms.py	Tue Dec 05 23:29:16 2023 +0000
+++ b/xpcom/ds/StaticAtoms.py	Tue Dec 05 23:58:06 2023 +0000
@@ -1627,6 +1627,7 @@
     Atom("onbeginEvent", "onbeginEvent"),
     Atom("onend", "onend"),
     Atom("onendEvent", "onendEvent"),
+    Atom("onrelease", "onrelease"),
     Atom("onrepeat", "onrepeat"),
     Atom("onrepeatEvent", "onrepeatEvent"),
     Atom("repeatCount", "repeatCount"),
